docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+version: '2.1'
+services:
+  web:
+    image: quay.io/mozmar/kitsune_base
+    build: .
+    command: ./manage.py runserver 0.0.0.0:8000
+    user: kitsune
+    volumes:
+      - ./:/app
+      - node_modules:/app/node_modules
+      - bower_components:/app/bower_components
+    depends_on:
+      - memcached
+      - mariadb
+      - elasticsearch
+      - redis
+    environment:
+      # Django settings overrides:
+      - DJANGO_SETTINGS_MODULE=kitsune.settings
+      - CELERY_CONFIG_MODULE=kitsune.settings
+      - ACCOUNT_DEFAULT_HTTP_PROTOCOL=http
+      - ALLOWED_HOSTS=*
+      - BROKER_HOST=rabbitmq
+      - BROKER_PORT=5672
+      - BROKER_USER=kitsune
+      - BROKER_PASSWORD=kitsune
+      - BROKER_VHOST=kitsune
+      - REDIS_URL=redis://redis:6379/0
+      - CELERY_ALWAYS_EAGER=False
+      - CSRF_COOKIE_SECURE=False
+      - DATABASE_URL=mysql://${DATABASE_USER:-root}:${DATABASE_PASSWORD:-kitsune}@mariadb:3306/${DATABASE_NAME:-kitsune}
```

Comment:
I don't see a use for that and it complicates things. We define a database to use and we already know the variables. There's no need to allow override those using env variables. If for some reason one wants to change database they can override `DATABASE_URL` entirely.